pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "great_expectations_cloud"
-version = "0.0.3.dev8"
+version = "0.0.3"
```

Comment:
This would make this the first version full/non-pre-release version.
I'm happy to change this back to a pre-release if we don't think we are ready.
But I wanted to get the logging changes into `great_expectations[cloud]` installs.